.github/actions/require-multiple-reviewers-v1/README.md

```diff
@@ -21,28 +20,15 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 2
     permissions:
-      pull-requests: write
+      pull-requests: read
       contents: read
+      checks: write
     steps:
```

Comment:
A checkout step of some sort is still required in order to read `important-files.txt`, no?